corehq/apps/export/tests/test_export_conversion.py

```diff
@@ -33,7 +34,13 @@
     CASE_HISTORY_TABLE,
 )
 
+MockRequest = namedtuple('MockRequest', 'domain')
 
+
+@mock.patch(
```

Comment:
this is somewhat regrettable and messy, but couldn't figure out a clean way to get around it